github/util.py

```diff
@@ -624,6 +624,9 @@ def retrieve_asset_contents(self, release_tag: str, asset_label: str):
         asset.download(buffer)
         return buffer.getvalue().decode()
 
+    def files_changed_in_pr(self, pr_number: str) -> typing.Generator[str, None, None]:
+        yield from (f.filename for f in self.repository.pull_request(pr_number).files())
```

Comment:
it should be equivalent to simply returning `.files()`, should it not? also: why not simply inline? apart from wrapping a generator in another generator, this method does not seem to do much, but pass-in pr_number